// mindee/src/product/fr/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.PayslipV2 = exports.IdCardV2 = exports.IdCardV1 = exports.EnergyBillV1 = exports.CarteVitaleV1 = exports.CarteGriseV1 = exports.BankAccountDetailsV2 = exports.BankAccountDetailsV1 = void 0;
var bankAccountDetailsV1_1 = require("./bankAccountDetails/bankAccountDetailsV1");
Object.defineProperty(exports, "BankAccountDetailsV1", { enumerable: true, get: function () { return bankAccountDetailsV1_1.BankAccountDetailsV1; } });
var bankAccountDetailsV2_1 = require("./bankAccountDetails/bankAccountDetailsV2");
Object.defineProperty(exports, "BankAccountDetailsV2", { enumerable: true, get: function () { return bankAccountDetailsV2_1.BankAccountDetailsV2; } });
var carteGriseV1_1 = require("./carteGrise/carteGriseV1");
Object.defineProperty(exports, "CarteGriseV1", { enumerable: true, get: function () { return carteGriseV1_1.CarteGriseV1; } });
var carteVitaleV1_1 = require("./carteVitale/carteVitaleV1");
Object.defineProperty(exports, "CarteVitaleV1", { enumerable: true, get: function () { return carteVitaleV1_1.CarteVitaleV1; } });
var energyBillV1_1 = require("./energyBill/energyBillV1");
Object.defineProperty(exports, "EnergyBillV1", { enumerable: true, get: function () { return energyBillV1_1.EnergyBillV1; } });
var idCardV1_1 = require("./idCard/idCardV1");
Object.defineProperty(exports, "IdCardV1", { enumerable: true, get: function () { return idCardV1_1.IdCardV1; } });
var idCardV2_1 = require("./idCard/idCardV2");
Object.defineProperty(exports, "IdCardV2", { enumerable: true, get: function () { return idCardV2_1.IdCardV2; } });
var payslipV2_1 = require("./payslip/payslipV2");
Object.defineProperty(exports, "PayslipV2", { enumerable: true, get: function () { return payslipV2_1.PayslipV2; } });
